content-test/lib/dedupe.test.js

```diff
@@ -1,13 +1,15 @@
-const {createDedupeKey, innerDedupe} = require("lib/dedupe");
 const {assert} = require("chai");
 const urlParse = require("url-parse");
 
+const dedupe = require("lib/dedupe");
```

Comment:
More tests for `fancy-dedupe` at https://github.com/k88hudson/fancy-dedupe/blob/master/test/index.test.js